app/assets/javascripts/common/password_validation.js

```diff
@@ -39,15 +30,41 @@ PasswordValidation = function() {
         "X-CSRF-Token": token
       },
       data: {"password": password}
-    }).done(function(data, status){
-      console.log(status)
-      console.log(data)
-      this.timer = null;
+    }).done((data, status) => {
+      let response;
```

Comment:
This is really `errors`, not the response...

Reply:
Done.